R/download_survey.r

```diff
@@ -33,7 +33,16 @@ download_survey <- function(survey, dir = NULL) {
     stop("'survey' is not a DOI or URL.")
   }
 
-  temp_body <- GET(url, config = list(followlocation = TRUE))
+  temp_body <- GET(
+    url,
+    config = config(
+      followlocation = 1,
+      useragent = paste0(
+        "http://github.com/epiforecasts/socialmixr R package socialmixr/v.",
+        packageVersion("socialmixr")
+      )
+    )
```

Comment:
`?httr::config` states:

> Generally you should only need to use this function to set CURL options directly if there isn't already a helpful wrapper function, like [set_cookies()](http://127.0.0.1:10145/help/library/httr/help/set_cookies), [add_headers()](http://127.0.0.1:10145/help/library/httr/help/add_headers) or [authenticate()](http://127.0.0.1:10145/help/library/httr/help/authenticate)

So I would recommend:

```suggestion
    config = config(
      followlocation = 1
    ),
    add_headers("user-agent" = paste0(
      "http://github.com/epiforecasts/socialmixr R package socialmixr/v.",
      packageVersion("socialmixr")
    ))
```

Not completely sure about the difference (some elements in [this SO question](https://stackoverflow.com/questions/52392262/what-is-different-between-set-opt-curlopt-useragent-and-set-useragent-in-header)) but it's probably safer to follow httr guidelines.

Reply:
ah thanks, I was struggling to combine them with the manual config but now see how it's done. In that case it would probably be even better to do:

```suggestion
    config = config(
      followlocation = 1,
      user_agent(paste0(
        "http://github.com/epiforecasts/socialmixr R package socialmixr/v.",
        packageVersion("socialmixr")
      ))
    )
```